python3/examples/pyproject.toml

```diff
@@ -3,4 +3,4 @@ requires = ["setuptools >= 38.6.0", "setuptools_scm[toml]", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
```

Comment:
You will need to update setup.cfg and probably remove this line:
```
[bdist_wheel]
# This flag says that the code is written to work on both Python 2 and Python
universal=1
```

Reply:
> You will need to update setup.cfg and probably remove this line:
> ```
> [bdist_wheel]
> # This flag says that the code is written to work on both Python 2 and Python 3
> universal=1
> ```
Thanks, removed.